lib/FIR/mixin.h

```diff
@@ -125,6 +114,12 @@ template<typename A, typename B> B &Unzip(B &out, A first, A last) {
       [](auto &&a) -> decltype(a.second) { return a.second; });
   return out;
 }
+
+template<typename A, typename B> B &UnzipSnd(B &out, A first, A last) {
```

Comment:
Can you add a comment like the ones on Zip and Unzip?

Also, I think the name should be spelled out, `UnzipSecond`.